### DISCUSSION TITLE
nb create到加载适配器时失败

### DISCUSSION THREAD

**Discussion by qiye2233** [nonebot/discussions#393] [问答]
![image](https://github.com/nonebot/discussions/assets/138650134/45622e18-1584-481e-aa4b-64ac495e3600)

**Comment by yanyongyu** [comment]
镜像获取都超时了，换个网或者挂代理

**Comment by whitedustmoon1175** [comment]
你试一下
pip install nonebot-adapter-cqhttp
我今天也这样

**Reply by yanyongyu** [reply]
cqhttp适配器早就不再使用了
